stories/breadcrumb.stories.js

```diff
@@ -1,5 +1,17 @@
 import { storiesOf } from '@storybook/html';
 import { withHits, withLifecycle } from '../.storybook/decorators';
+import { connectHierarchicalMenu } from '../src/connectors';
+import { noop } from '../src/lib/utils';
+
+const virtualHierarchicalMenu = (args = {}) =>
+  connectHierarchicalMenu(noop, noop)({
```

Comment:
The widget is hidden so I could have used a `configure` widget, but at the moment it's doesn't work. The widget uses `setQueryParameters` which means that it will override the previous value. We have to update the behavior of the widget to use `mergeSearchParameters` (in a separate PR). It will ease the migration for our users since we advocate a lot for the usage of `configure`.

Reply:
you mean that the logic here:

https://github.com/algolia/instantsearch.js/blob/bf0497864d880e41007e80ac163d270fcc3f83a0/src/connectors/configure/connectConfigure.ts#L145

should be like here:

https://github.com/algolia/instantsearch.js/blob/bf0497864d880e41007e80ac163d270fcc3f83a0/src/connectors/configure/connectConfigure.ts#L94

?

Reply:
Yes.

```js
getWidgetSearchParameters(state, { uiState }) {
  return mergeSearchParameters(
    state,
    new algoliasearchHelper.SearchParameters({
      ...uiState.configure,
      ...widgetParams.searchParameters
    })
  );
}
```